frontend-react/src/__mocks__/WatersMockServer.ts

```diff
@@ -70,6 +70,8 @@ const watersResponseError = {
 
 const handlers = [
     rest.post(`${RS_API_URL}/api/waters`, (req, res, ctx) => {
+        if (req.headers["_headers"]["client"] === "test-endpoint-name")
```

Comment:
Can we add "test-endpoint-name", bad-client", etc... into an enum

Reply:
Oo, good call! Updated 👍🏼